prow/plugins/cla/cla.go

```diff
@@ -93,20 +92,20 @@ type gitHubClient interface {
 }
 
 func handleStatusEvent(pc plugins.Agent, se github.StatusEvent) error {
-	return handle(pc.GitHubClient, pc.Logger, se)
+	return handle(pc.GitHubClient, pc.Logger, se, pc.PluginConfig.CLAConfig)
 }
 
 // 1. Check that the status event received from the webhook is for the CNCF-CLA.
 // 2. Use the github search API to search for the PRs which match the commit hash corresponding to the status event.
 // 3. For each issue that matches, check that the PR's HEAD commit hash against the commit hash for which the status
 //    was received. This is because we only care about the status associated with the last (latest) commit in a PR.
 // 4. Set the corresponding CLA label if needed.
```

Comment:
nit: could you update this comment to reflect the new cla logic? I think only point `1` would need updating